servers/rendering_server.h

```diff
@@ -1684,7 +1710,7 @@ class RenderingServer : public Object {
 
 #ifndef DISABLE_DEPRECATED
 	// Never actually used, should be removed when we can break compatibility.
-	enum Features {
+	enum Features{
```

Comment:
This change looks unintentional

Reply:
I think that is just `clang-format` doing its thing?

Reply:
Hm, it's going the opposite direction of our coding standards, though. If you add the space back, does `clang-format` really remove it?

Reply:
It seems to be a clang-format bug. It's a bit iffy when it comes to heavy usage of macros.

Removing the `#else` stuff added at the top of this file "fixes" the format bug:
```diff
diff --git a/servers/rendering_server.h b/servers/rendering_server.h
index 240d82c90b..a16bfa70af 100644
--- a/servers/rendering_server.h
+++ b/servers/rendering_server.h
@@ -43,6 +43,7 @@
 
 // Helper macros for code outside of the rendering server, but that is
 // called by the rendering server.
+
 #ifdef DEBUG_ENABLED
 #define ERR_ON_RENDER_THREAD                                              \
 	RenderingServer *rendering_server = RenderingServer::get_singleton(); \
@@ -60,11 +61,6 @@
 	RenderingServer *rendering_server = RenderingServer::get_singleton(); \
 	ERR_FAIL_NULL_V(rendering_server, m_ret);                             \
 	ERR_FAIL_COND_V(!rendering_server->is_on_render_thread(), m_ret);
-#else
-#define ERR_ON_RENDER_THREAD
-#define ERR_ON_RENDER_THREAD_V(m_ret)
-#define ERR_NOT_ON_RENDER_THREAD
-#define ERR_NOT_ON_RENDER_THREAD_V(m_ret)
 #endif
 
 template <typename T>
@@ -1710,7 +1706,7 @@ public:
 
 #ifndef DISABLE_DEPRECATED
 	// Never actually used, should be removed when we can break compatibility.
-	enum Features{
+	enum Features {
 		FEATURE_SHADERS,
 		FEATURE_MULTITHREADED,
 	};
```

I managed to make a MRP by cutting down most content:
```cpp
#ifdef DEBUG_ENABLED
// Remove #else to fix formatting bug.
#else
#endif

class RenderingServer : public Object {
#ifndef DISABLE_DEPRECATED
	enum Features{
		FEATURE_SHADERS,
		FEATURE_MULTITHREADED,
	};
#endif
};
```

I'll file a bug report upstream.

For now, we just have to accept the bug.

Reply:
@akien-mga Thanks for digging into this!

I guess for the purposes of this PR, we can just ignore the style issue.

Reply:
Yeah we kinda need that else block :) I could just fix it correctly and than put a clang-ignore block around it for now?

Reply:
No I would just keep it as is. Then one day the upstream bug may be fixed and this will be fixed automatically.